index.html

```diff
@@ -18,6 +18,8 @@
 <body>
   <div id="app"></div>
   <script type="module" src="/src/main.ts"></script>
+  <script src="https://unpkg.com/vue@3"></script>
```

Comment:
@HuuTai33 Why did you re-import _vue3_ and _hcaptcha/vue3-hcaptcha_ from unpkg? They seem to be already included in the final bundle.

This version of the vue3 also corresponds to a development version... it doesn't seem optimal to me. If you check the dev console on ecocode.io, the following message appears:
> You are running a development build of Vue.
> Make sure to use the production build (*.prod.js) when deploying for production.

Reply:
oh yes my bad, will fix it !

Reply:
Fixed in #6